packages/graphiql/package.json

```diff
@@ -54,6 +54,7 @@
     "@graphiql/toolkit": "^0.8.0",
     "entities": "^2.0.0",
     "graphql-language-service": "^5.1.0",
+    "regenerator-runtime": "0.13.11",
```

Comment:
added as dependency since imported in https://github.com/graphql/graphiql/blob/6c3ca902d769893a4201f8856299af18b7e21a1b/packages/graphiql/src/cdn.ts#L10

Reply:
removed as in this PR https://github.com/graphql/graphiql/pull/2939